composer/loggers/logger_hparams.py

```diff
@@ -8,6 +8,7 @@
 from dataclasses import asdict, dataclass
 from typing import TYPE_CHECKING, Any, Dict, List, Optional
 
+import coolname
```

Comment:
can you move the coolname import into ``initialize_object``, since it is a conditional install?